runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md

```diff
@@ -0,0 +1,30 @@
+# KubevirtHyperconvergedClusterOperatorCRModification
+
+## Meaning
+
+Hyperconverged Cluster Operator configures clusters in an opinionated way and overwrites its operands when there is an unexpected change to them. Users are expected to not modify the operands directly. The `HyperConverged` object is the source of truth for the configuration.
```

Comment:
- I'm not sure if the term _configures clusters_ is accurate. Maybe _configures kubevirt and its supporting operators_ ?
- the HyperConverged object --> the HyperConverged *custom resource*

Reply:
done